rswag-specs/lib/rswag/specs/openapi_formatter.rb

```diff
@@ -164,6 +130,116 @@ def remove_invalid_operation_keys!(value)
         value.delete(:request_examples) if value[:request_examples]
       end
 
+      def parse_parameters(doc)
+        doc[:paths]&.each_pair do |_k, path|
+          path.each_pair do |_verb, endpoint|
+            is_hash = endpoint.is_a?(Hash)
+            if is_hash && endpoint[:parameters]
+              mime_list = endpoint[:consumes] || doc[:consumes]
+              parse_endpoint(endpoint, mime_list) if mime_list
+            end
+            remove_invalid_operation_keys!(endpoint)
+          end
+        end
+      end
+
+      def parse_endpoint(endpoint, mime_list)
+        parameters = endpoint[:parameters]
+        # But there can be any number of formData
+        parameters.select { |p| p[:schema] if p[:in] == :formData  }.each do |schema_param|
+          parse_parameter_schema(endpoint, schema_param, mime_list)
```

Comment:
@stevegeek, I'm testing the changes in https://github.com/RedHatInsights/compliance-backend/compare/master...romanblanco:compliance-backend:rswag-v3 and have a problem that response examples are not being generated.

The `parse_parameter_schema` method is never called, so nor is the `set_mime_examples`.

Reply:
Investigating issues with `RSWAG_DRY_RUN`, `after` callbacks are not being triggered.